src/sonic_ax_impl/mibs/ietf/rfc3433.py

```diff
@@ -424,6 +425,16 @@ def update_xcvr_dom_data(self):
 
             # skip RJ45 port
             transceiver_info_entry_data = Namespace.dbs_get_all(self.statedb, mibs.STATE_DB, mibs.transceiver_info_table(interface))
+    
+            if 'type' not in transceiver_info_entry_data:
+                # Only write error log once
+                if interface not in self.broken_transceiver_info:
+                    mibs.logger.error(
```

Comment:
>error [](http://example.com/codeflow?start=32&length=5)

log at warning level, because it is tolerable to snmpagent.

Reply:
Fixed